### ISSUE TITLE
ShapeNet Evaluation Results

### ISSUE THREAD

**Issue by QingkaiLu** [ShivamDuggal4/TARS3D#1]
Hi Shivam,

I have ran the evaluation for the airplane and car categories of ShapeNet. The reconstructed meshes are quite impressive. I have two questions regarding the ShapeNet evaluation. 

(1) For my evaluation on ShapeNet, the loss of car is 1.837e-02, the chamfer distance of car is 0.0124|0.0240. The loss of airplane is 2.289e-02, the chamfer distance of car is 0.0182|0.0160. The chamfer distances of my evaluation are smaller than that reported in the paper. For example, the chamfer distance of airplane is 0.194|0.152 in the paper. Could you please let me know what might cause the differences? 

(2) I have to change the name of the ground truth point cloud from "pointcloud3.npz" to "pointcloud.npz" in this [file](https://github.com/ShivamDuggal4/TARS3D/blob/master/dataloader/shapenet.py#L141) for ShapeNet, since the name of the ShapeNet pointcloud is "pointcloud.npz" instead of ""pointcloud3.npz". Is this fine? 

Thank you very much!

**Comment by ShivamDuggal4** [comment]
Hi @QingkaiLu 
Thanks for your warm words and your interest in our work! Please find the answers to your questions below:

* The shared models were retrained post paper submission and after code cleaning. The slight difference in metrics could have come from the little training changes made during cleaning, for eg: by changing the number of training examples for which the additional point features were randomly turned off during stage 2 training. 
* `pointcloud3.npz` refers to the processed version of `pointcloud.npz`. To download these post-processed point clouds, please refer to [this](https://github.com/chenhsuanlin/signed-distance-SRN#shapenet) and [SDF-SRN supplementary section B.1](https://arxiv.org/pdf/2010.10505.pdf).

Hope these answer your questions!

Best Regards
Shivam

**Comment by QingkaiLu** [comment]
Thank you for the explanation, Shivam. That is quite helpful.
